src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/CSharp.xaml

```diff
@@ -61,4 +61,9 @@
   <StringProperty Name="LastGenOutput" Visible="false" Description="The filename of the last file generated as a result of the SFG." />
   <BoolProperty Name="DesignTime" Visible="false" Description="A value indicating whether this file has a designer." />
   <BoolProperty Name="AutoGen" Visible="false" Description="A value indicating whether this is a generated file." />
+  <StringProperty Name="CustomTool" Visible="false" Description="DTE Property for accessing the Generator property.">
```

Comment:
Please give a better description, I know you copied this from None, but this is a user visible string.